helix-term/src/ui/mod.rs

```diff
@@ -265,6 +266,76 @@ pub fn file_picker(root: PathBuf, config: &helix_view::editor::Config) -> FilePi
     picker
 }
 
+pub fn file_browser(root: PathBuf) -> Result<FilePicker, std::io::Error> {
+    let root = root.canonicalize()?;
```

Comment:
```suggestion
    let root = helix_stdx::path::canonicalize(root);
```

generally we try to avoid the `canonicalize` from `std` since it resolves symlinks

Reply:
I just copied the code from the file_picker here =) But you are right, we don't want to resolve symlinks in the file browser.